PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSubJetFraction.h

```diff
@@ -139,7 +142,8 @@ class AliAnalysisTaskSubJetFraction : public AliAnalysisTaskEmcalJet {
   Int_t                               fSoftDropOn;
 
   Bool_t                              fNsubMeasure;
-  
+  TF1                                 *kT_Func;
+  TF1                                 *Omega_Func;
```

Comment:
Not needed if the functions stay in local scope.

Reply:
Hi Markus,

Initially I declared them in the function and then used the delete operator when they were done, but it gave me an error "incomplete declaration of type TF1". So I was playing around a bit to fix this and I forgot to move them back. I will send a new patch.

Thanks,
Nima

Reply:
Hi @nzardosh ,

Normally there should no be an error when you declare the variables in the function. I guess this is a compiler error, right? This usually means you are missing a header file to be included.

Please leave the pull request open and push the patch to this pull request!

Cheers

Markus

Reply:
Hi Markus,

Yes I had to add the header for TF1 to make it work before. I will attach the update to this pull request but since its too late for todays tag now I will add a few other things first and then add them all together.

Thanks,
Nima